interface-definitions/include/interface/vif.xml.i

```diff
@@ -22,25 +22,29 @@
     <leafNode name="egress-qos">
       <properties>
         <help>VLAN egress QoS</help>
-        <completionHelp>
-          <script>echo Format for qos mapping, e.g.: '0:1 1:6 7:6'</script>
-        </completionHelp>
+        <valueHelp>
+          <format>from:to</format>
+          <description>The format is FROM:TO with multiple mappings separated by spaces.</description>
+        </valueHelp>
         <constraint>
-          <regex>[:0-7 ]+$</regex>
+          <regex>[:0-7]+$</regex>
         </constraint>
-        <constraintErrorMessage>QoS mapping should be in the format of '0:7 2:3' with numbers 0-9</constraintErrorMessage>
+        <constraintErrorMessage>QoS mapping should be in the format of '0:7' with numbers 0-9</constraintErrorMessage>
+        <multi />
```

Comment:
If multiple values are supported by a space delimiter, why use a multi node?

Reply:
The reason why null values are not used is to ensure format consistency. When a user uses more than one space in a string, adding or removing a space will trigger remove. In fact, there is no need to change the VLAN wrapper interface